sys/event/Kconfig

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_EVENT
+    bool "Event queue"
+    depends on TEST_KCONFIG
+    select MODULE_CORE_THREAD_FLAGS
+    help
+        This module offers an event queue framework like libevent or libuev.
+
+        An event queue is basically a FIFO queue of events, with some functions
+        to efficiently and safely handle adding and getting events to / from
+        such a queue.
+
+if MODULE_EVENT
+
+config MODULE_EVENT_CALLBACK
+    bool "Support for callback-with-argument event type"
+
+menuconfig MODULE_EVENT_THREAD
+    bool "Support for event handler threads"
+    help
+        There are three threads of different priorities that can be enabled.
+
+if MODULE_EVENT_THREAD
+
+config MODULE_EVENT_THREAD_LOW
```

Comment:
I just noticed that the actual name of this symbol should be `MODULE_EVENT_THREAD_LOWEST`, will push a fix

Reply:
Done